code/frameworks/nextjs/src/preset.ts

```diff
@@ -73,10 +77,63 @@ export const previewAnnotations: StorybookConfig['previewAnnotations'] = (entry
   require.resolve('@storybook/nextjs/preview.js'),
 ];
 
-// export const webpackInstance = async () => {
-//   webpack.init();
-//   return webpack;
-// };
+// Not even sb init - automigrate - running dev
+// You're using a version of Nextjs prior to v10, which is unsupported by this framework.
```

Comment:
is this comment supposed to be here?

Reply:
Yes. I have readded this whole section, because it was removed in the initial branch.